kmwine02/project01
# Modified the artist.html and js

To test:
 
- When user searches new artist, new card should replace previous search
- When user searches new artist, button should appear below search box
